client/src/components/SocialMediaShow.js

```diff
@@ -10,32 +10,35 @@ const SocialMediaShow = (props) => {
     description: "",
   });
 
-  const [comments, setComments] = useState([]);
+  const [comments, setComments] = useState([])
+  const [newComment, setNewComment] = useState({ rating: "", comment: "" })
```

Comment:
this state doesn't look like it's being used? the CommentForm defines a very similar piece of state. With forms, we ideally want the state for the `<form>` to be defined in the same component